resource-sharing-partners/src/test/java/no/sikt/rsp/ResourceSharingPartnerTest.java

```diff
@@ -180,11 +180,13 @@ public void shouldLogExceptionWhenS3ClientFails() {
     }
 
     @Test
-    public void shouldLogExceptionWhenS3BucketFileCannotBeConvertedToBaseBibliotek() throws IOException {
+    public void shouldSkipWhenCannotBeConvertedToBaseBibliotek() throws IOException {
```

Comment:
Mangler her ikke en tet med minst to librrayCodes i en Liste hvor den første skipes og den andre konverteres?
Så at vi ender opp med `expectedSuccessfulConversion=1` og `expectedFaieledConversion=1`.

Reply:
fikset

Reply:
Jeg ser ikke helt hvordan det addresserer min spørsmål. 
Kanskje jeg var ikke tydelig nok.
Jeg mener, vi burde har en fil på s3-bucket med minst 2 bibNr inn i. Den første skulle feile på en måte, mens den andre bli prosessert likevel. Dermed kunne vi teste om iterasjon over alle bibNr på s3 ikke knekke hele kjøringer.
Dere har nå 2 tester. Begge tester oppererer bare med et bibnr.
Eller har jeg misforstodd det.